h5py_wrapper/wrapper.py

```diff
@@ -266,10 +258,10 @@ def _load_dataset(f, lazy=False):
         if value_type == 'NoneType':
             return None
         else:
-            if (len(f.attrs.keys()) > 0 and
-                    'custom_shape' in f.attrs.keys()):
+            if (len(list(f.attrs.keys())) > 0 and
```

Comment:
Actually, the list conversion is not needed here. 
We can just leave the code to be `len(f.attrs.keys())`.